packages/tldraw/src/lib/shapes/shared/usePrefersReducedMotion.tsx

```diff
@@ -5,6 +5,7 @@ export function usePrefersReducedMotion() {
 	const [prefersReducedMotion, setPrefersReducedMotion] = useState(false)
 
 	useEffect(() => {
+		if (typeof window === undefined || !('matchMedia' in window)) return
```

Comment:
looks like there's some copy-pasta bug here in the codebase - pushing up a fix to this PR.

should be: `typeof window === 'undefined'` not literally `undefined`